backend/codesirius/authentication/serializers/signup.py

```
@@ -19,10 +19,14 @@ class SignupSerializer(serializers.Serializer):
             "min_length": "First name is too short.",
             "max_length": "First name is too long.",
         },
+        source="first_name",
+    )
+    lastName = serializers.CharField(
+        source="last_name", required=False, allow_blank=True
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)